orchagent/aclorch.cpp

```diff
@@ -3596,6 +3596,7 @@ sai_status_t AclOrch::createDTelWatchListTables()
     if (status != SAI_STATUS_SUCCESS)
     {
         SWSS_LOG_ERROR("Failed to create table %s", flowWLTable.description.c_str());
+        handleSaiCreateStatus(SAI_API_ACL, status);
```

Comment:
>handleSaiCreateStatus [](start = 8, length = 21)

Check the return value? If successfully handled, you treat it as success?

Reply:
There are a few SAI calls that do not take any action in the case of SAI failures. For these ones, since our goal is to make it on-par with async mode at this stage, I am not checking the return value for now so that it has the same behavior as before.

Reply:
Let's assume this function magically retry and fix the issue, so the return value is `task_success`. You should return `SAI_STATUS_SUCCESS` after this function.

---
In reply to: [595609272](https://github.com/Azure/sonic-swss/pull/1665#discussion_r595609272) [](ancestors = 595609272)

Reply:
Agree, updated the logic to check if the return value is `task_success`, and proceed when it is.